tests/drivers/sensor/sbs_gauge/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+set(CONF_FILE "proj.conf")
+
+if(${BOARD} STREQUAL "nucleo_f070rb")
+  set(DTC_OVERLAY_FILE
+    "${DTC_OVERLAY_FILE} ${CMAKE_CURRENT_LIST_DIR}/dts/overlays/nucleo_f070rb.overlay"
+    CACHE STRING "" FORCE
+    )
+endif()
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(integration)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+zephyr_include_directories(include)
```

Comment:
```suggestion
```

Copy/pasted from somewhere? There's no include directory here.

Reply:
Did you mean to respond to this one? Marking unresolved as I think the question stands.